tests/gdimagefill/bug00104_1.c

```diff
@@ -0,0 +1,40 @@
+/* $Id$ */
+#include "gd.h"
+#include <stdio.h>
+#include "gdtest.h"
+
+int main()
```

Comment:
this testcase doesn't look like the one you posted in the commit message.  is this what you meant to include ?